peerconnection.go

```diff
@@ -829,6 +829,17 @@ func (pc *PeerConnection) startRTPReceivers(incomingTracks map[uint32]trackDetai
 		remoteIsPlanB = descriptionIsPlanB(pc.RemoteDescription())
 	}
 
+	// Ensure we haven't already started a transcevier for this ssrc
```

Comment:
```suggestion
	// Ensure we haven't already started a transceiver for this ssrc
```

Reply:
Fixed!

Reply:
@audiolion would you be interested in joining the Pion org! You would be be able to approve PRs, and then would be able to merge your own (once approved)

No real obligation, but always looking to grow the community :)

Reply:
@Sean-Der sure that would be cool! I will try to help out in any way I can.